src/webgl/p5.RendererGL.js

```diff
@@ -637,6 +637,13 @@ p5.RendererGL.prototype.strokeJoin = join => {
   console.error('Sorry, strokeJoin() is not yet implemented in WEBGL mode');
 };
 
+p5.RendererGL.prototype.filter = filterType => {
+  // filter can be achieved using custom shaders.
+  // https://github.com/aferriss/p5jsShaderExamples
+  // https://itp-xstory.github.io/p5js-shaders/#/
```

Comment:
Can you move this comment along with a note that filter does not work in WebGL mode to the user-facing reference for filter. This comment is more for the user than the developer, and if you add it to [this section ](https://github.com/processing/p5.js/blob/master/src/image/pixels.js#L354)it will appear in the reference docs.